tests/core_tests.py

```diff
@@ -155,8 +156,9 @@ def test_slices_V2(self):
                 (slc.slice_name, 'slice_url', slc.slice_url),
             ]
         for name, method, url in urls:
-            print("[{name}]/[{method}]: {url}".format(**locals()))
+            logging.info("[{name}]/[{method}]: {url}".format(**locals()))
             response = self.client.get(url)
+        self.logout()
```

Comment:
ah, forgot to log out that different user. curious that it passed on my branch. thanks for the fix!